lms/services/__init__.py

```diff
@@ -27,20 +27,24 @@
 
 def includeme(config):
     config.register_service_factory(
-        "lms.services.h_api_requests.HAPIRequests", name="h_api_requests"
+        "lms.services.application_instance_getter.ApplicationInstanceGetter",
```

Comment:
I alphabetized these service registrations. Hope that's not controversial.

Reply:
:+1:

I've been thinking, by the way, about how each of our services in both h and lms has two different names. For example `"ApplicationInstanceGetter"` and `"ai_getter"`. I wonder about just using the class name for the service names, so it'd be `request.find_service(name="ApplicationInsanceGetter")` rather than `request.find_service(name="ai_getter")`. Just to make the code easier to grep.

Or we could use interfaces, like `request.find_service(IApplicationInstanceGetter)`.